recipes/calysto_bash/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "calysto_bash" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4e3869f7c86a534085deb88bc6af6f41d1b5e105638f8958398a62029996510f
+
+build:
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - {{ PYTHON }} -m calysto_bash install --sys-prefix
+
+requirements:
+  host:
+    - pip
+    - python
+    - ipykernel
+    - metakernel
+  run:
+    - python
+    - ipykernel
+    - jupyter_client
+    - metakernel
+    - bash  # [unix]
+    - m2-bash  # [win]
+
+test:
+  imports:
+    - calysto_bash
+  commands:
+    - jupyter kernelspec list | grep bash
+    - jupyter run --kernel=bash command.sh
```

Comment:
The kernel is called `calysto_bash`: https://github.com/Calysto/calysto_bash/blob/dfa6833187e1fe9d9c229fd7bcc839fd7813d74b/calysto_bash/kernel.json#L8

Reply:
Sorry. Fixed.